config/locales/en/claims/schools/claims.yml

```diff
@@ -34,6 +34,8 @@ en:
           mentors: Mentors
           page_title: Claim - %{reference}
           hours_of_training: Hours of training
+          grant_funding: Grant funding
           hours: hours
+          claim_amount: £%{amount}
```

Comment:
Given that you are using `Money.new` you should be able to use the helpers within the gem https://github.com/RubyMoney/money-rails?tab=readme-ov-file#helpers, so this locale should be unnecessary